### DISCUSSION TITLE
talos install

### DISCUSSION THREAD

**Discussion by wsy0101** [siderolabs/sidero#1211] [General]
Hello, I encountered a confusion while deploying a cluster for Sidero metal. I hope you can provide me with an answer. The description of the problem is as follows:

I have deployed a set of sidero metal in the K8S environment on the public network. Can this sidero metal perform talos installation on bare metal in the private network (a local area network that can access the public network)?

**Comment by smira** [comment]
yes, it can, Sidero Metal is designed for a private data center scenarios.

It might be better to use discussions for questions.

**Reply by wsy0101** [reply]
We are currently building one such environment in the laboratory. Sidero metal is deployed on a public network, and we hope to install Talos on the private LAN through Sidero metal. This private LAN can ping to the public network, but the public network cannot ping to the private LAN. All of this is based on IPv4 communication. The problem we are currently facing is that Talos cannot bootstrap ETCD when deployed on devices in the private LAN, Is there a solution for this?
